docs/kb/laptop-dev-env.md

```diff
@@ -0,0 +1,224 @@
+# NovaCustom laptop development environment
+
+This document describes the process of configuring a development and debugging
+environment for NovaCustom laptops.
+
+The instruction is applicable for workstations running Fedora. If you're using
+a different distro, some commands (e.g. package installation) may be different.
+
+## Embedded Controller
+
+This section describes the development environment for EC firmware on NovaCustom
+devices. The Embedded Controller firmware is heavily based on System76 EC
+firmware and makes use of open-source debugging tools and software also
+developed by System76.
+
+### Flashing
+
+This section outlines the parts and software needed to perform external flashing
+of the Embedded Controller.
+
+| Size | 14 inch models | 16 inch models |
+| --- | --- | --- |
+| Programmer | Arduino Mega 2560 | Arduino Mega 2560 |
+| Cable | **FFC 0.5mm pitch 24 pin** | **FFC 1.0mm pitch 24 pin** |
```

Comment:
What about the adapter from the FFC to arduino pin header?

Grounding cable?
USB-B to USB-A for Arduino?

Reply:
Added the rest of the cables to the table: https://github.com/Dasharo/docs/compare/4e9ec5622c840b8a6d840a400efafdd5de3ee80f..6a19fdc3b9c3ed99815a29ce364e8e90543a5ac2

Reply:
I still don't see the adapter on the list. Currently it would be impossible to connect the FFC cable to arduino.